### PR TITLE
19 technical implement lti connection for moodle

### PR DESCRIPTION
**Pull request by theUpsider**

### CODE REVIEW

File: setup/Table_Setup.sql
Comment (MarcNor) [inline setup/Table_Setup.sql:1]: Same here, should this be out?

File: tests/unit/lti/test_OIDCLoginFlask.py [hunk 1]
```diff
@@ -0,0 +1,315 @@
...
+from unittest.mock import MagicMock, patch
+
+from errors import errors as err
+from service_layer.crypto import JWTKeyManagement
+from service_layer.lti.config.ToolConfigJson import ToolConfigJson
+from service_layer.lti.OIDCLoginFlask import OIDCLoginFlask
+
+# ignore E501
+config_file = {
+    "https://moodle.haski.app": {
```
Comment (MarcNor) [inline tests/unit/lti/test_OIDCLoginFlask.py:13]: here as well, shouldn't that be a env var?

**Review by MarcNor — APPROVED** [review]
In general good, maybe think about comments

File: entrypoints/HASKI-OAS.yaml [hunk 2]
```diff
@@ -196,53 +196,7 @@ paths:
             application/json:
               schema:
                 $ref: "#/components/schemas/GeneralError"
-  /user/{user_id}/{lms_user_id}/contactform:
```
[the remark this answers is not in the document]
Comment (theUpsider) [reply entrypoints/HASKI-OAS.yaml:199]: now it is i hope

**Pull request merged**